openlibrary/catalog/marc/test_marc.py

```diff
@@ -1,27 +1,56 @@
 #!/usr/bin/python
 
 import unittest
-from parse import normalize_isbn, biggest_decimal, compile_marc_spec
+from get_subjects import subjects_for_work
+from marc_base import MarcBase
+from parse import  read_isbn, read_pagination, read_title
 
 class MockField:
     def __init__(self, subfields):
         self.subfield_sequence = subfields
         self.contents = {}
         for k, v in subfields:
             self.contents.setdefault(k, []).append(v)
-    def get_elts(self, i):
-        if i in self.contents:
-            return self.contents[i]
-        else:
-            return []
-class MockRecord:
-    def __init__(self, subfields):
+
+    def get_contents(self, want):
```

Comment:
The argument `want` is a little unclear to me; would `subfield_code` be better? Same applies to the other methods in this class. A docstring might also be useful; `get_contents` and `get_subfields` sound like the could do the same thing.

Reply:
If you feel inclined to update these, would happily support @cdrini's comment -- unto itself the meaning of want v. subfields is slightly confusing.

Reply:
@cdrini, @mekarpeles I was stepping through trying to understand and parallel the live code here, which is where I copied the `want` argument name from.  I'll review the original MARC classes (binary and XML) and add some docstrings there because they have none, and could really benefit from some too!